src/ripple/rpc/Role.h

```diff
@@ -52,13 +55,12 @@ enum class Role
 */
 Role
 requestRole (Role const& required, Port const& port,
```

Comment:
the implementation of this function still has the old signature (thus the no-unity build failures)

Reply:
fixed